src/js/vre/chapter36/config/form.js

```diff
@@ -88,30 +99,72 @@ const formConfig = {
               }
             }
           }
+        },
+        nonVeteranInformation: {
+          title: 'Applicant information',
+          path: 'non-veteran-information',
```

Comment:
LMK if I need to update this path. I've seen other patterns follow a `chapter/page` format.

Reply:
@stacyhlinn Do you have a good name for this page? We can use that come up with the url.

Reply:
If only dependents can claim this (besides the Veteran), could we call this dependent information?

Reply:
@stacyhlinn is out but I like the idea to update it to dependent-information and will do that. We can update it later if we need to.